parity-util-mem/Cargo.toml

```diff
@@ -34,7 +36,7 @@ optional = true
 
 [features]
 default = ["std", "ethereum-impls"]
-std = ["parking_lot"]
+std = ["parking_lot", "lru", "hashbrown"]
```

Comment:
If I'm not mistaken, both `hashbrown` and `lru` are no-std libs.

Reply:
indeed, std bound can be removed, nice catch!